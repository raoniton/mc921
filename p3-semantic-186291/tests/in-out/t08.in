class Main {

    public void f() {
        char a = 'a';
        char b = 'b';
        return;
    }

    public void f() {
        return;
    }

    public static void main(String[] args) {
        this.f();
    }
}