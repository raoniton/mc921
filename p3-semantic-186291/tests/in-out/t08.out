SemanticError: f method has already been defined @ 9:5
